torch/_dynamo/variables/builder.py

```diff
@@ -853,6 +856,43 @@ def wrap_tensor(self, value: torch.Tensor):
 
         return tensor_variable
 
+    def wrap_numpy_ndarray(self, value):
+        assert isinstance(value, np.ndarray)
+        arg_proxy = self.tx.output.root_tracer.create_graph_input(
+            re.sub(r"[^a-zA-Z0-9]+", "_", self.name), type(value)
+        )
+        source = self.get_source()
+        tensor_value = torch.from_numpy(value)
+
+        from ..utils import to_torch_np_ndarray
+
+        numpy_ndarray_variable = wrap_fx_proxy_cls(
+            target_cls=NumpyNdarrayVariable,
+            tx=self.tx,
+            proxy=self.tx.output.create_proxy(
+                "call_function",
+                to_torch_np_ndarray,
+                (arg_proxy,),
+                {},
+            ),
+            example_value=tensor_value,
+            source=source,
+        )
+        self.tx.output.input_source_to_var[source] = numpy_ndarray_variable
+        example_value = numpy_ndarray_variable.proxy.node.meta["example_value"]
+
+        grapharg = GraphArg(
+            source,
+            tensor_value,
+            False,
+            example_value,
+            is_tensor=True,
+            example_strong_ref=tensor_value,
```

Comment:
We had better be careful about this, it is easy to end up accidentally leaking tensors this way. See https://github.com/pytorch/pytorch/pull/100515